### ISSUE TITLE
Sample Math on filters is wrong

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio-frontend#227]
The counts on the filters is double the actual number of samples

![screen shot 2018-08-07 at 3 31 57 pm](https://user-images.githubusercontent.com/15315514/43798079-714442c2-9a57-11e8-8b9e-014ada7f47a9.png)

It should show 30 samples in the filters.

**Comment by Willv19** [comment]
The number displayed on the frontend is what is returned from the backend. I will open a ticket in the backend for the fix.

**Comment by jaclyn-taroni** [comment]
Depends on: https://github.com/AlexsLemonade/refinebio/issues/509

**Comment by arielsvn** [comment]
I talked to @dvenprasad regarding that question, and she said the main goal of users is to find samples. That's why it's better to have the number of samples on the filters.

**Comment by Miserlou** [comment]
I can't reproduce this, maybe it was confusion about samples/experiment count?

![screen shot 2018-08-23 at 11 54 09 am](https://user-images.githubusercontent.com/139987/44536815-4c17cc00-a6cb-11e8-986f-88b325ec9a43.png)

?

**Comment by cgreene** [comment]
It's possible something else fixed this. @dvenprasad's screenshot does seem to show the wrong numbers. @dvenprasad : can you still reliably reproduce this?

**Comment by arielsvn** [comment]
Me neither, this probably was also fixed by https://github.com/AlexsLemonade/refinebio-frontend/pull/230

Before that, the user can end up in a state where the results didn't correspond with the active filters.

**Comment by dvenprasad** [comment]
No I can't reproduce this either now. 
But why is the 'Add to Dataset' button disabled? Could be a separate issue)

**Comment by arielsvn** [comment]
> But why is the 'Add to Dataset' button disabled? Could be a separate issue)

Yes, that's related to https://github.com/AlexsLemonade/refinebio/issues/529. Right now we only allow processed samples into the dataset, but that information isn't available in the search page.

Talked to @Miserlou about maybe modifying the dataset endpoint, to allow sending only the experiment id instead of all the sample ids.

**Comment by arielsvn** [comment]
ok I'm closing this, we can reopen it if the bug appears again
